sklearn/model_selection/_validation.py

```diff
@@ -406,7 +406,7 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     X : array-like of shape at least 2D
         The data to fit.
 
-    y : array-like, optional, default: None
+    y : array-like
```

Comment:
the default should still be there, and probably the shape as well

Reply:
@adrinjalali I will definitely add the shape. However, I'm a bit confused about the default value for y on _fit_and_score: I noticed inconsistencies with how the X and y were given in _fit_and_score (no default for y) and cross_validate (y=None) when defining the function. I matched the docs to accurately reflect each parameter that had a default value in the function definition. Do you suggest that I should make my docs more consistent, even if that means not quite describing the code's setup exactly? Or should I add a keyword argument (y=None) for _fit_and_score and change the docs to match it? This is my first time contributing, so all insights are helpful :) Thanks.

Reply:
I wouldn't touch the code here. If there's not default, you don't have to mention it.

and you're right, this one doesn't have a default, it's my bad. You can instead simply have it as `array-like or None`. We don't have to be too picky about these ones since they're private functions anyway.